### ISSUE TITLE
one practice?

### ISSUE THREAD

**Issue by bbkali** [himadriganguly/sqlilabs#2]

**Comment by himadriganguly** [comment]
Can you please elaborate on what is the issue?

**Comment by bbkali** [comment]
Hello, there is no more code for practical exercises in the project, only one exercise?example1.php？

**Comment by himadriganguly** [comment]
Yeah. Mainly I am not getting the time to contribute to the project. If you like the project, please contribute to the project.
